cmd/loop/lsat.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/lightninglabs/loop/looprpc"
+	"github.com/lightninglabs/loop/lsat"
+	"github.com/urfave/cli"
+	"gopkg.in/macaroon.v2"
+)
+
+type printableToken struct {
+	ID              string `json:"id"`
+	ValidUntil      int64  `json:"valid_until"`
+	BaseMacaroon    string `json:"base_macaroon"`
+	PaymentHash     string `json:"payment_hash"`
+	PaymentPreimage string `json:"payment_preimage"`
+	AmountPaid      int64  `json:"amount_paid_msat"`
+	RoutingFeePaid  int64  `json:"routing_fee_paid_msat"`
+	TimeCreated     int64  `json:"time_crated"`
```

Comment:
typo: I'm guessing should be `time_created`

Reply:
Yes, thanks!